ot/gromov.py

```diff
@@ -572,6 +574,349 @@ def df(G):
         return log['fgw_dist']
 
 
+def GW_distance_estimation(C1, C2, p, q, loss_fun, T, number_sample=[-1, -1], std=True):

```

Comment:
number_sample should be None by default, default with a  test in the function,  lists in functuions is a big source of Bugs. Same it should be None instead of -1 if nothing is given (dont' forget to upadte the doc also).

Reply:
Also what number_samples correpond to is unclear the first and second have very different impact, maybe split them in two with more understandable names?